docker/Dockerfile.deps

```diff
@@ -21,8 +21,8 @@ RUN rm -f /opt/python/cp27-cp27m
 # Boost
 RUN BOOST_VERSION=1.66.0 && \
     cd /usr/local && \
-    curl -L https://dl.bintray.com/boostorg/release/1.66.0/source/boost_${BOOST_VERSION//./_}.tar.gz | tar -xzf - && \
-    ln -s ../boost_${BOOST_VERSION//./_}/boost include/boost
+    curl -L https://github.com/boostorg/boost/archive/boost-${BOOST_VERSION}.tar.gz | tar -xzf - && \
+    ln -s /boost-${BOOST_VERSION}/boost include/boost
```

Comment:
Just curious: why? Has the official repo moved or sth?

Reply:
no idea, but I cannot access it anymore.

Reply:
while I was rebuilding the deps image multiple times, it stopped working from one day to the next and then never came back since. I talked to Janusz and he favors replacing it with the GitHub boost archive.